docs/hugo/content/contributing/add-a-new-code-generated-resource/implement-extensions.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Customize the behaviour of generated resource
```

Comment:
Do we want some (if needed) suffix here?

Or maybe just a disclaimer at the start that many resources don't need any extensions defined?

Reply:
Added disclaimer.